pallets/parachain-staking/src/weights.rs

```diff
@@ -814,58 +780,26 @@ impl WeightInfo for () {
 	// Storage: ParachainStaking Total (r:1 w:1)
 	#[rustfmt::skip]
 	fn execute_delegator_bond_less() -> Weight {
-		Weight::from_ref_time(93_139_000 as u64)
+		Weight::from_ref_time(97_377_000 as u64)
 			.saturating_add(RocksDbWeight::get().reads(8 as u64))
 			.saturating_add(RocksDbWeight::get().writes(8 as u64))
 	}
 	// Storage: ParachainStaking DelegatorState (r:1 w:1)
 	// Storage: ParachainStaking DelegationScheduledRequests (r:1 w:1)
 	#[rustfmt::skip]
 	fn cancel_revoke_delegation() -> Weight {
-		Weight::from_ref_time(39_815_000 as u64)
+		Weight::from_ref_time(43_406_000 as u64)
 			.saturating_add(RocksDbWeight::get().reads(2 as u64))
 			.saturating_add(RocksDbWeight::get().writes(2 as u64))
 	}
 	// Storage: ParachainStaking DelegatorState (r:1 w:1)
 	// Storage: ParachainStaking DelegationScheduledRequests (r:1 w:1)
 	#[rustfmt::skip]
 	fn cancel_delegator_bond_less() -> Weight {
-		Weight::from_ref_time(46_787_000 as u64)
+		Weight::from_ref_time(49_020_000 as u64)
 			.saturating_add(RocksDbWeight::get().reads(2 as u64))
 			.saturating_add(RocksDbWeight::get().writes(2 as u64))
 	}
-	// Storage: MoonbeamOrbiters ForceRotation (r:1 w:0)
-	// Storage: ParachainStaking Points (r:1 w:0)
-	// Storage: ParachainStaking Staked (r:1 w:2)
-	// Storage: ParachainStaking InflationConfig (r:1 w:0)
-	// Storage: ParachainStaking ParachainBondInfo (r:1 w:0)
-	// Storage: System Account (r:302 w:301)
-	// Storage: ParachainStaking CollatorCommission (r:1 w:0)
-	// Storage: ParachainStaking CandidatePool (r:1 w:0)
-	// Storage: ParachainStaking TotalSelected (r:1 w:0)
-	// Storage: ParachainStaking CandidateInfo (r:9 w:0)
-	// Storage: ParachainStaking DelegationScheduledRequests (r:9 w:0)
-	// Storage: ParachainStaking TopDelegations (r:9 w:0)
-	// Storage: ParachainStaking AutoCompoundingDelegations (r:9 w:0)
-	// Storage: ParachainStaking Total (r:1 w:0)
-	// Storage: ParachainStaking AwardedPts (r:2 w:1)
-	// Storage: ParachainStaking AtStake (r:1 w:10)
-	// Storage: MoonbeamOrbiters OrbiterPerRound (r:1 w:0)
-	// Storage: MoonbeamOrbiters CurrentRound (r:0 w:1)
-	// Storage: ParachainStaking SelectedCandidates (r:0 w:1)
-	// Storage: ParachainStaking DelayedPayouts (r:0 w:1)
-	#[rustfmt::skip]
-	fn round_transition_on_initialize(x: u32, y: u32, ) -> Weight {
-		Weight::from_ref_time(363_268_000 as u64)
-			// Standard Error: 1_140_000
-			.saturating_add(Weight::from_ref_time(43_560_000 as u64).saturating_mul(x as u64))
-			// Standard Error: 3_000
-			.saturating_add(Weight::from_ref_time(139_000 as u64).saturating_mul(y as u64))
-			.saturating_add(RocksDbWeight::get().reads(180 as u64))
-			.saturating_add(RocksDbWeight::get().reads((4 as u64).saturating_mul(x as u64)))
-			.saturating_add(RocksDbWeight::get().writes(171 as u64))
-			.saturating_add(RocksDbWeight::get().writes((1 as u64).saturating_mul(x as u64)))
-	}
```

Comment:
Is this benchmark now split into several small ones?

Reply:
This benchmark was wrong, explanations here: https://github.com/PureStake/moonbeam/pull/1893